packages/server/src/api/muster/muster.postgres.controller.it.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import { expectNoErrors } from '../../util/test-utils/expect';
+import { seedAll } from '../../util/test-utils/seed';
+import { TestRequest } from '../../util/test-utils/test-request';
+import { User } from '../user/user.model';
+import { Unit } from '../unit/unit.model';
+
+
+describe('Muster Controller Compliance', () => {
+  const basePath = '/api/muster';
+  let req: TestRequest;
+  let seedData: any[];
+  let orgId: number;
+  let unitId: number;
+
+  beforeEach(async () => {
+    seedData = await seedAll();
+    req = new TestRequest(basePath);
+    req.setUser(await User.findOne());
+
+    orgId = seedData[0].org.id;
+    unitId = seedData[0].units.filter(((u: Unit) => u.name === 'Unit 1' && u.org!.id === orgId)).map((unit: { id: any }) => unit.id)[0];
+  });
+
+  it('should return calculated compliance', async () => {
+    const fromDate = '2020-01-01T00:00:00.000';
+    const toDate = '2020-01-14T23:00:00.000';
+
+    const res = await req.get(`/${orgId}/roster?fromDate=${fromDate}Z&toDate=${toDate}&unitId=${unitId}&page=0&limit=10`);
+
+    expectNoErrors(res);
+    expect(res.data)
+      .is
+      .eql(
+        { rows: [
+          { totalMusters: 10,
```

Comment:
Do we really need all this user information coming back? Like how does phone number and custom column for example even matter for the return of this calculation?

Reply:
This is the original payload returned by the ES API. There was no change to the UI code at all.
<img width="1605" alt="Screen Shot 2021-08-12 at 9 42 37 AM" src="https://user-images.githubusercontent.com/58491416/129394478-d1291838-a250-4bc0-8ab6-296a36f129c4.png">
 The user information is displayed on the UI screen together with the calculations.

Reply:
Got it, but I still do not see custom columns? IDK the payload may have just contained that initially due to how things have to be organized in ES.